src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudServiceActor.scala

```diff
@@ -1,228 +0,0 @@
-package org.broadinstitute.dsde.firecloud
-
-import akka.stream.ActorMaterializer
-import org.broadinstitute.dsde.firecloud.dataaccess._
-import org.broadinstitute.dsde.firecloud.elastic.ElasticUtils
-import org.broadinstitute.dsde.firecloud.metrics.MetricsActor
-import org.broadinstitute.dsde.firecloud.model.{ModelSchema, UserInfo, WithAccessToken}
-import org.slf4j.LoggerFactory
-import spray.http.StatusCodes._
-import spray.http._
-import spray.routing.{HttpServiceActor, RequestContext, Route}
-import org.broadinstitute.dsde.firecloud.service._
-import org.broadinstitute.dsde.firecloud.webservice._
-import org.broadinstitute.dsde.workbench.util.health.HealthMonitor
-import org.elasticsearch.client.transport.TransportClient
-
-import scala.concurrent.duration._
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.language.postfixOps
-
-
-class FireCloudServiceActor extends HttpServiceActor with FireCloudDirectives
```

Comment:
as noted above, this class has been removed and replaced with `FireCloudApp.scala` and `FireCloudApiService.scala`